### PR TITLE
Instructor Course Details Page shows table header with no body if there are no students #3933

### LINKED ISSUE #3933: Instructor Course Details Page shows table header with no body if there are no students
Display "No students enrolled" instead of table header when no students are enrolled

screen shot 2015-07-09 at 1 07 51 pm

### PR DESCRIPTION
**Pull request by bamboo3250**
Fix #3933

@damithc Does this look good to you?

empty table

### CODE REVIEW

**Comment by damithc** [comment]
Check with others. We have a similar 'empty table' in other places (i've forgotten how they look). We should follow the same here, for consistency.

**Comment by bamboo3250** [comment]
I found a lot of empty tables without a body. 

No Active Course

emptyactivecourse

No feedback session

emptysession

No course in Instructor Home Page

emptyinstructorhome

And includes No student in Instructor Course Details above. None of them has a body when empty. There are Ongoing PRs like #2847 but it seems not completed.

**Comment by yj-soh** [comment]
I believe studentHome does already show something when there are no sessions; you can refer to it.

**Comment by taniach** [comment]
screen shot 2015-07-09 at 5 01 47 pm

Student home shows a message if the table is empty. We can include messages in other empty tables including Instructor Course Details. Having a status message looks better than having empty body panel or table header without body.

**Comment by taniach** [comment]
We can add a link to an action along with the message. Example:

No students enrolled. Click here to enroll students.

**Comment by damithc** [comment]
Looks like we are not very consistent in how we show the empty table. I prefer the one that sows an empty row with empty cells together with a yellow status message.

**Comment by bamboo3250** [comment]
This is what it looks like now. I will push the code when finishing running the test.

newemptytable

**Comment by bamboo3250** [comment]
@yj-soh Could you review this for me?

File: src/main/java/teammates/ui/controller/InstructorCourseDetailsPageAction.java [hunk 1]
```diff
@@ -49,6 +49,10 @@ public ActionResult execute() throws EntityDoesNotExistException {
         
         data.init(instructor, courseDetails, instructors, students);
         
+        if (students.size() == 0) {
```
Comment (yj-soh) [inline src/main/java/teammates/ui/controller/InstructorCourseDetailsPageAction.java]: `isEmpty()`?

**Comment by bamboo3250** [comment]
@yj-soh ok, I fixed it

**Comment by yj-soh** [comment]
Looks okay, make sure it passes the tests.

**Pull request merged**